io.github.shiftey.Desktop.yaml

```
@@ -0,0 +1,95 @@
+app-id: io.github.shiftey.Desktop
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: start-github-desktop
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

I didn't say to add it if Wayland was not supported.

---

my bad